lhotse/testing/dummies.py

```diff
@@ -34,14 +34,27 @@ def dummy_recording(unique_id: int) -> Recording:
         duration=1.0
     )
 
+def dummy_alignment(symbol: str = "irrelevant", start: float = 0.0, duration: float = 1.0) -> AlignmentItem:
+    return AlignmentItem(
+        symbol=symbol,
+        start=start,
+        duration=duration
+    )
 
-def dummy_supervision(unique_id: int, start: float = 0.0, duration: float = 1.0, text: str = "irrelevant") -> SupervisionSegment:
+def dummy_supervision(
+    unique_id: int,
+    start: float = 0.0,
+    duration: float = 1.0,
+    text: str = "irrelevant",
+    alignment: Optional[Dict[str, List[AlignmentItem]]] = None
```

Comment:
I suggest to use `dummy_alignment()` as the default value, which will let you leverage a lot of existing unit tests for this